src/client/builtin_scene/instanced_mesh.cpp

```diff
@@ -436,7 +427,7 @@ namespace builtin_scene
     return len * sizeof(InstanceData);
   }
 
-  void RenderableInstancesList::beforeInstancedDraw(WebGL2Context &glContext, CSSBorderDataTexture *borderDataTexture)
+  void InstanceListBase::beforeInstancedDraw(WebGL2Context &glContext, CSSBorderDataTexture *borderDataTexture)
```

Comment:
The `beforeInstancedDraw` and `afterInstancedDraw` can be virtual, because `borderDataTexture` is only used by the content instances list.